packages/dcos-integration-test/extra/integration_test.py

```diff
@@ -970,7 +970,8 @@ def test_octarine_http(cluster, timeout=30):
         'id': '/integration-test-app-octarine-http-{}'.format(test_uuid),
         'cpus': 0.1,
         'mem': 128,
-        'ports': [10001],
+        'ports': [0],
+        'requirePorts': True,
```

Comment:
I was thinking "why are we even bothering to hand-specify a port, we should let Marathon pick one out which is non-conflicting for us rather than maintain a list manually", and have been iterating. Was likely going to drop the requirePorts and just do the `'ports': [0]` shortly.